rmw_zenoh_cpp/src/detail/event.hpp

```diff
@@ -53,17 +51,7 @@ enum rmw_zenoh_event_type_t
 /// Helper value to indicate the maximum index of events supported.
 #define ZENOH_EVENT_ID_MAX rmw_zenoh_event_type_t::ZENOH_EVENT_PUBLICATION_MATCHED
```

Comment:
Should we move this to the translation unit too?

Reply:
As it stands, this is still needed by graph_cache.cpp.  I don't see a good way to remove this right now, so I'd prefer to leave it.

Reply:
Sounds good.